src/WebJobs.Script/Environment/EnvironmentSettingNames.cs

```diff
@@ -37,6 +37,7 @@ public static class EnvironmentSettingNames
         public const string EnvironmentNameKey = "AZURE_FUNCTIONS_ENVIRONMENT";
         public const string FunctionsV2CompatibilityModeKey = "FUNCTIONS_V2_COMPATIBILITY_MODE";
         public const string EasyAuthEnabled = "WEBSITE_AUTH_ENABLED";
+        public const string StaticWebAppFunction = "STATIC_WEB_APP_FUNCTION";
```

Comment:
@cgillum this can be changed easily, but it's something we would have to add to system app settings so that users can't set that app setting themselves.

Reply:
after talking to Annaji, it seems we add the environment variable "BLUERIDGE"=true on all the static web apps functions, so that seems like the most painless way to distinguish if a function belongs to static web apps or not. Any other name is doable, but will involve backfilling all the existing function apps.

Reply:
The problem with `BLUERIDGE` is that not all blueridge deployments are fronted my MSHA, so it's unsafe to rely on this value.

Reply:
I think the inverse is also true. With BYO functions, we might see "regular" function apps behind MSHA.